tests/e2e/firestore/firestore.e2e.js

```diff
@@ -96,6 +96,30 @@ describe('firestore()', () => {
       );
     });
 
+    it('should reject invalid cache sizes', async () => {
+      try {
+        await firebase.firestore().settings({ cacheSizeBytes: '100' });
+      } catch (error) {
+        return Promise.resolve();
+      }
+
+      return Promise.reject(
+        new Error('applied settings with cacheSizeBytes lower then minimal.')
```

Comment:
I think this test is verifying a string value throws an error, but the error message isn't correct?

Reply:
As long as you're in there - just because I think it will be easy? Can you add a check that the -1/unlimited size is allowed through and doesn't error? it'll complement the minimum check that follows this one

Reply:
Fixed the error message to be related to the test.

I've tried to add this case too but it causes settings to be set on the native site which blocks. I suppose, for this reason, the tests in the settings() block are all testing just the validation part.

Reply:
ah, okay , thanks for checking